### DISCUSSION TITLE
Crawl and monitor full website

### DISCUSSION THREAD

**Discussion by rahulkumarsingh73690** [upptime/upptime#905] [Ideas] [answered]
Hi,

Is it possible to monitor the entire website for errors (e.g., 500)? Sometimes, due to a code issue, the website homepage works; however, other pages are down. The uptime monitor can't detect this because it only tracks the homepage.

**Comment by AnandChowdhary** [comment, marked as answer]
Currently, you have to hardcode and specify a list of URLs to monitor. There is no way in Upptime to crawl all the pages on a website and monitor automatically.
